src/components/DocumentCard/DocumentCardTitle.scss

```diff
@@ -1,9 +1,12 @@
 @import '../../common/common';
 
+$documentCard-titleSize: 16px;
```

Comment:
In Fabric we have 17px(size-l) or 15px(size-m-plus) defined, but no 16px. This case is an abnormally where we have to use 16px.